packages/abi-gen/src/utils.ts

```diff
@@ -92,4 +93,24 @@ export const utils = {
             inputs: [],
         };
     },
+    makeOutputFileName(name: string): string {
+        let fileName = toSnakeCase(name);
+        if (fileName === 'z_r_x_token') {
+            fileName = 'zrx_token';
+        }
+        return fileName;
+    },
+    writeOutputFile(filePath: string, renderedTsCode: string): void {
+        fs.writeFileSync(filePath, renderedTsCode);
+    },
+    isOutputFileUpToDate(abiFile: string, outputFile: string): boolean {
+        const abiTime = fs.statSync(abiFile).mtimeMs;
+        try {
+            const outputFileTime = fs.statSync(outputFile).mtimeMs;
+            return outputFileTime > abiTime;
+        } catch (e) {
```

Comment:
Our convention is to use variable name `err` for errors. `catch (err) {`

Reply:
Let's explicitly check for the `ENOENT` error here and return `false` if it is thrown. For all other errors, we should re-throw it. E.g `Permission denied` errors (see all: https://nodejs.org/api/errors.html#errors_common_system_errors)